examples/tutorials/Generating_molecules_with_MolGAN.ipynb

```diff
@@ -10,7 +10,7 @@
         "\n",
```

Comment:
<p>Can we add a comment below about why we see so many failures?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>